ios/FluentUI/Vnext/List/List.swift

```diff
@@ -106,16 +143,24 @@ extension MSFListView {
     struct MSFListCellView: View {
         var cell: MSFListVnextCellData
         var layoutType: MSFListCellVnextLayoutType
+        @State var isExpanded: Bool
```

Comment:
I think this should be @ObservedObjcect var state: MSFListVnextCellData as you need all of the cell behavior in sync with the object data.